### PR TITLE
doc-review-qs copy edit of quick starts for ModelMesh

### PR DESCRIPTION
**Pull request by MelissaFlinn**
#### Motivation
Copy edit requested by development

#### Modifications
Multiple updates to the readme files

#### PR checklist

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Unit tests pass locally
- [ ] FVT tests pass locally
- [ ] If the PR adds a new container image or updates the tag of an existing image (not build within cpaas), is the corresponding change made in live-builder and cpaas-midstream to add/update the image tag in the operator CSV? Link the PRs if applicable

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Tested modelmesh serving deployment with odh-manifests and ran odh-manifests-e2e tests locally

### CODE REVIEW

**Comment by MelissaFlinn** [comment]
Needs a review by @Jooho

**Comment by MelissaFlinn** [comment]
My apologies - this PR needs a review by Jooho Lee before it is merged and so I changed the status to DRAFT

File: opendatahub/docs/get-started-odh-modelserving.md
Comment (Jooho) [inline opendatahub/docs/get-started-odh-modelserving.md:7]: Just found that `or dedicated-admin` needed to be removed. please remove this part.

File: opendatahub/quickstart/README.md
Comment (Jooho) [inline opendatahub/quickstart/README.md:11]: Sample Model Deployment and Persistent Volume Claim would be like this
Sample Model Deployment using Persistent Volume Claim

File: opendatahub/quickstart/common_docs/modelmesh-install.md
Comment (Jooho) [inline opendatahub/quickstart/common_docs/modelmesh-install.md:22]: This folder can be changeable so I think changing it to this would be better. What do you think?
```
cd opendatahub/quickstart/$QUICK_START_FOLDER   # (ex) cd opendatahub/quickstart/basic  
```

File: opendatahub/quickstart/pvc/README.md
Comment (Jooho) [inline opendatahub/quickstart/pvc/README.md:1]: sync with the title in another doc.
~~~
# Quick Start - Sample Model Deployment using Persistent Volume Claim
~~~

**Comment by Jooho** [comment]
@MelissaFlinn Thanks for your review!! I leave some comments on the files. Please take a look at them.
Plus, could you add your signature to the commit. This is how you can do
~~~
oc commit -s -m "XXX"
~~~

**Comment by MelissaFlinn** [comment]
Thank you @Jooho - I addressed your comments and added my signature to the second commit.
This PR is ready for merging after approval by the project owners. Thank you

**Review by Jooho — APPROVED** [review]
/lgtm

**Pull request merged**
